bin/compile

```diff
@@ -37,8 +37,8 @@ case "${VERSION}" in
   3) VERSION="3.2.13";;
   4) VERSION="4.0.14";;
   5) VERSION="5.0.14";;
-  6) VERSION="6.2.11";;
-  7) VERSION="7.0.9";;
+  6|6.2) VERSION="6.2.12";;
```

Comment:
It looks like there's no way to get redis 6.0 with this setup. But maybe that was the case before, too?

Reply:
That issue was introduced here: https://github.com/heroku/heroku-buildpack-ci-redis/commit/6182a830691210123b2ce1996f4aca9e0915d4ed

Ideally, this should match what's supported for the add-on: 4, 5, 6.0, 6.2, and 7. In a few months, after the deprecation is done, we should remove 4, 5, and 6.0.